documentation/general/dash-sonic-hld.md

```diff
@@ -273,20 +274,34 @@ dst_port                 = list of range of destination ports ',' separated
 
 ### 3.2.5 ROUTING TYPE
 	
+Two different schemas are used for the routing type table. The first is used when there is only a single action associated with a routing type:
 ```
-DASH_ROUTING_TYPE_TABLE:{{routing_type}}: [
-        "action_name":{{string}}
-        "action_type": {{action_type}} 
+DASH_ROUTING_TYPE_TABLE:{{routing_type}}
+        "action_name": {{string}}
+        "action_type": {{action_type}}
         "encap_type": {{encap type}} (OPTIONAL)
         "vni": {{vni}} (OPTIONAL)
-    ]
+
+key                      = DASH_ROUTING_TYPE_TABLE:routing_type:action_name; routing type can be {direct, vnet, vnet_direct, vnet_encap, appliance, privatelink, privatelinknsg, servicetunnel}; action_name is a string
+; field                  = value
+action_name              = action name as a string
+action_type              = action_type can be {maprouting, direct, staticencap, appliance, 4to6, mapdecap, decap, drop}
+encap_type               = encap type depends on the action_type - {vxlan, nvgre}
+vni                      = vni value associated with the corresponding action. Applicable if encap_type is specified. 
 ```
 
+The second schema is used when multiple, ordered actions are associated with a routing type:
 ```
-key                      = DASH_ROUTING_TYPE_TABLE:routing_type; routing type can be {direct, vnet, vnet_direct, vnet_encap, appliance, privatelink, privatelinknsg, servicetunnel}; actions can be a list of action_types
+DASH_ROUTING_TYPE_TABLE:{{routing_type}}:{{action_name}}
+        "action_type": {{action_type}}
```

Comment:
Further changes will be needed here since we cannot guarantee that all actions for a routing type will be programmed at the same time - there may be a delay between them which need to be handled before we push the updates to SAI